js/index.js

```diff
@@ -0,0 +1,217 @@
+  innovation.innerHTML = "Innovation";  // added innovation to a innerHtml
+  innovation.href = "#"; // add href to a tag
+  headerNav.appendChild(innovation); // header nav appendchild innovation
+
+  let home = document.createElement("a");
+  home.innerHTML = "Home";
+  home.href = "#";
+  headerNav.prepend(home);
+
+  headerNavA = document.querySelectorAll("header nav a");  // select all a in the header nav to a node list
+  headerNavA.forEach(navItem => {     // iterate through nav nodelist
+    navItem.style.color = "green";    // each item reasisgn the color of a to green
+  });
+
+  //Stretch task starts here.
+  ctaButton.style.backgroundColor = "blue";
+  ctaButton.style.color = "white";
+
+  let body = document.querySelector("body");
+  body.style.backgroundColor = "pink";
+
+  ctaH1.style.fontSize = "100px";
+  ctaH1.style.color = "white";
+  ctaH1.style.textShadow = "5px 5px black";
+
+  setInterval(function(){            // added a setInterval to toggle color green and white every second
+    if(ctaH1.style.color === "green") {
+      ctaH1.style.color = "white";
+    } else {
+    ctaH1.style.color = "green";
+    }
+  }, 1000);
+
+  // DOM II starts here
+
+ctaButton.addEventListener("click", function(){  // added click eventListener to the cta button
+  TweenMax.from(ctaImg, 3, {
+    marginLeft: 400,
+    rotation: 720,
+    scale: 0.3
+  });
+});
+
+let contactBtn = document.querySelector(".contact .contact-btn");
+
+contactBtn.addEventListener("click", function() {
+  TweenMax.to(contactH4, 1, {color: "red", opacity: 1, x: "40%"});
+  TweenMax.to(contactAddress, 1, {color: "red", opacity: 1, x: "40%"});
+  TweenMax.to(contactPhone, 1,{color: "red", opacity: 1, x: "40%"});
+  TweenMax.to(contactEmail, 1, {color: "red", opacity: 1, x: "40%"});
+  TweenMax.to(contactBtn, 1, {opacity: 0});
+});
+
+copyRight.addEventListener("mouseover", function() {
+  TweenMax.to(copyRight, 2, {color: "blue", scale: 2, padding: 10, marginBottom: 50});
+});
+
+imgLogo.addEventListener('mouseenter', function(){ //added mouse enter eventListener to the img logo
+  TweenMax.from(imgLogo, 3, {
+    x:400,
+    ease:Back.easeOut
+  });
+});
+
+
+ctaImg.addEventListener('dblclick', function(){ // added dblclick eventListener to the cta img
+  TweenMax.staggerFrom(headerNavA, 3, {
+    opacity:0,
+    y:200,
+    rotation:360});
+});
+
+ctaH1.addEventListener('click', function(){ //click eventListener to "DOM is Awesome"
+  let topContentH4 = document.querySelectorAll(".main-content .top-content .text-content h4");
+  TweenMax.from(topContentH4, 3, {
+    x:-400,
+    ease:Back.easeOut});
+});
+
+
+let topContentP = document.querySelectorAll(".main-content .top-content .text-content p"); //having the top content move from the right
+TweenMax.from(topContentP, 3, {
+  x:400,
+  ease:Back.easeOut
+});
+
+
+middleImg.addEventListener('mouseover', function(){ //mouseover eventListener to middle img
```

Comment:
Interesting... if you keep mousing over this middle image, it gets dimmer and eventually goes away. I might have to spend some time playing with this one :smiley: